### ISSUE TITLE
Sign Call SC operation feature

### ISSUE THREAD

**Issue by longgamma-massa** [massalabs/station-massa-wallet#587]
**Context**
Currently we give no context when a dapp is requesting a signature for CallSC operation. 
Current issue is a **nice-to-have priority** for the Bridge ICO scope.

<img width="540" alt="image" src="https://github.com/massalabs/station-massa-wallet/assets/113043914/a8c939ce-2af6-48aa-8b78-9eae531c7f0e">

**User flow**
- User is requesting to sign a CallSC operation in Massa wallet that was called via other dApp connected (Bridge)
- User should understand key info about the operation he is about to sign. 
Info to provide:
- Initiator ('From' address/account) hash 
- Contract hash
- Operation type: CallSC (in this case)
- Gas (full decimals following the units standard)
- Transaction fee (full decimals in XMA, following units standard)
- https://github.com/massalabs/station-massa-wallet/issues/588
- Description provided by DApp #556 which should ideally provide the called function (method)

**How to**
- [designs](https://www.figma.com/file/LSaOibVDFb1reF6FePVn5l/Massastation?type=design&node-id=5237-27232&mode=design&t=mX2n36EmUpo1kKx1-0) (need validation after tech eval)

**Technical details**

- [x] https://github.com/massalabs/station-massa-wallet/issues/588
- [x] https://github.com/massalabs/station-massa-wallet/issues/597
- [x] https://github.com/massalabs/station-massa-wallet/issues/556
- [x] #607

**QA testing**
*Does this task require some QA tests ?*
*If yes, explain how to validate it*

**Comment by longgamma-massa** [comment]
@B-Naoufal please assign ETA and action plan before starting

**Comment by longgamma-massa** [comment]
@mazmassa Here's the ticket for Signing operation on Massa wallet. You should sync with @B-Naoufal as he did the backend and basic frontend implementation

**Comment by B-Naoufal** [comment]
> @mazmassa Here's the ticket for Signing operation on Massa wallet. You should sync with @B-Naoufal as he did the backend and basic frontend implementation

@mazmassa call me whenever you're available so we can sync !

**Comment by pivilartisant** [comment]
All tasks of the issue were completed
